courses/sync_external_courses/emeritus_api_test.py

```diff
@@ -33,6 +34,39 @@
 from mitxpro.utils import clean_url
 
 
+@pytest.fixture
+def emeritus_course_json():
+    """
+    Emeritus Course JSON with Future dates.
+    """
+    start_date = (datetime.now() + timedelta(days=1)).strftime("%Y-%m-%d")  # noqa: DTZ005
+    end_date = (datetime.now() + timedelta(days=2)).strftime("%Y-%m-%d")  # noqa: DTZ005
+    return {
+        "program_name": "Internet of Things (IoT): Design and Applications",
+        "course_code": "MO-DBIP",
+        "course_run_code": "MO-DBIP.ELE-25-07#1",
+        "start_date": start_date,
+        "end_date": end_date,
+        "Category": "Technology",
+        "list_price": 2600,
+        "list_currency": "USD",
+        "total_weeks": 7,
+        "product_family": "Certificate",
+        "product_sub_type": "Short Form",
+        "format": "Online",
+        "suggested_duration": 49,
+        "language": "English",
+        "landing_page_url": "https://test-emeritus-api.io/Internet-of-things-iot-design-and-applications"
+        "?utm_medium=EmWebsite&utm_campaign=direct_EmWebsite?utm_campaign=school_website&utm_medium"
+        "=website&utm_source=MIT-web",
+        "Apply_now_url": "https://test-emeritus-api.io/?locale=en&program_sfid=01t2s000000OHA2AAO&source"
+        "=applynowlp&utm_campaign=school&utm_medium=MITWebsite&utm_source=MIT-web",
+        "description": "Test Description",
+        "learning_outcomes": None,
+        "program_for": None,
```

Comment:
nit: We have similar data in https://github.com/mitodl/mitxpro/blob/master/courses/sync_external_courses/test_data/batch_test.json. Would it be good to read the course from the json file and use that as a sample course for our fixtures. This way will be able to remove the duplicate code and it would be easier if there is a change in the schema of the external course data.